### ISSUE TITLE
Add Support 12.0.2 

### ISSUE THREAD

**Issue by ghost** [Scandal-UK/Incognito_RCM#14]
![IMG_1849](https://user-images.githubusercontent.com/83787307/118745684-fe6a2700-b856-11eb-8441-7747cf63b580.JPG)
Incognito_Rcm does not work on firmware 12.0.2 and atmospehre 19.3 please help me for support

**Comment by 20Phoenix** [comment]
Same with newest FW 12.1.0  😢

**Comment by greensad** [comment]
Yes please, if only to restore my backup so that I can switch to exosphere
